src/transformers/__init__.py

```diff
@@ -775,6 +777,13 @@
             "LayoutLMModel",
         ]
     )
+    _import_structure["models.layoutlmv2"].extend(
+        [
+            "LAYOUTLMv2_PRETRAINED_MODEL_ARCHIVE_LIST",
+            "LayoutLMv2ForTokenClassification",
+            "LayoutLMv2Model",
```

Comment:
It could be beneficial to also add a `LayoutLMv2ForSequenceClassification`, which you can use to classify document images. It would probably be very similar to `LayoutLMForSequenceClassification`.

Reply:
For sequence classification, the paper describes  a method leveraging visual embeddings... beyond just classifying [CLS] output. To add similar behavior, the base layout model will also requires some additional changes to propagate those embeddings. See: https://github.com/microsoft/unilm/issues/329